src/daily_output.ts

```diff
@@ -243,33 +243,38 @@ export class DailyOutput {
 			throw new Error('That block was already processed!');
 		}
 
-		const queryTimestamp = edge.node.block.timestamp * 1000;
-		const queryDate = new Date(queryTimestamp);
-		const previousTimestamp = this.latestTimestamp * 1000;
-		const previousDate = new Date(previousTimestamp);
-
-		const daysDiff = daysDiffInEST(previousDate, queryDate);
-		for (let index = 0; index < daysDiff; index++) {
-			this.resetDay();
-		}
-
-		const weeksDiff = weeksDiffInEST(previousDate, queryDate);
-		for (let index = 0; index < weeksDiff; index++) {
-			this.resetWeek();
-		}
-
 		const fee = node.fee.winston ? +node.fee.winston : undefined;
 		const tip = node.quantity.winston ? +node.quantity.winston : undefined;
 		// we are using EPSILON here to have a minimum range of error acceptance
 		const isTipValid = await validateTxTip(node, this.ardriveOracle);
 		const entityTypeTag = tags.find((tag) => tag.name === 'Entity-Type')?.value;
 		const bundleVersion = tags.find((tag) => tag.name === 'Bundle-Version')?.value;
 		const isMetadataTransaction = !!entityTypeTag && !bundleVersion;
+		const isFileMetadata = entityTypeTag === 'file';
 		const isBundleTransaction = !!bundleVersion;
-
 		const bundledIn = node.bundledIn?.id;
+		const isV2DataTx = !isMetadataTransaction && !isBundleTransaction && !bundledIn;
+
+		if (fee && tip && isTipValid && (isV2DataTx || isBundleTransaction)) {
```

Comment:
We could get rid of `fee && tip` if we made `validateTxTip` to return `false` when `fee` is zero. It already returns false if `tip` is zero